package.json

```diff
@@ -6,11 +6,15 @@
 		"@testing-library/jest-dom": "^5.11.10",
 		"@testing-library/react": "^11.2.6",
 		"@testing-library/user-event": "^12.8.3",
+		"bootstrap": "^4.6.0",
```

Comment:
Are we using bootstrap/reactstrap components?

Reply:
I will be messing with cards from Reactstrap and if it doesn't do what I want I will be removing those :)

Reply:
Awesome sounds good!